server/model/article.js

```diff
@@ -54,11 +60,11 @@ export class ArticleFactory {
     // Annoyingly Wordpress doesn't always send the featured image over with the attachments,
     // so we revert to the thumbnail.
     const mainImage: ?Picture = getWpFeaturedImage(json.featured_image, json.attachments) || thumbnail;
-    const mainMedia: Array<Video | Picture> = [mainImage, mainVideo].filter(Boolean);
+    const mainMedia: Array<Video | Picture> = [mainImage, mainVideo, mainComic].filter(Boolean);
```

Comment:
To avoid polluting the model with an arbitrary `isMain`, we could pollute this bit (which is the parser) with:
```JS
const mainMedia: Array<Video | Picture> = [(mainComic || mainImage), mainVideo].filter(Boolean);
```

?